src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BitcoinActivationRule.cs

```diff
@@ -23,31 +23,10 @@ public override Task RunAsync(RuleContext context)
         {
             Guard.NotNull(context.ConsensusTip, nameof(context.ConsensusTip));
 
-            BlockHeader header = context.ValidationContext.Block.Header;
+            BlockHeader header = context.ValidationContext.ChainedHeader.Header;
 
             int height = context.ConsensusTipHeight + 1;
 
-            // Check proof of work.
-            if (header.Bits != context.NextWorkRequired)
```

Comment:
@dangershony good job moving these to separate rules! NICE.

Reply:
Seemed logical